source/SIL.AppBuilder.Portal/src/routes/(authenticated)/directory/+page.svelte

```diff
@@ -17,45 +17,64 @@
   let productDefinitionFilter: string;
   let organizationFilter: string;
   let updateDates: [Date, Date] | null;
-</script>
 
-<ProjectSelector
-  projects={data.projects.filter((proj) => {
+  let searchTerm: string = '';
```

Comment:
As far as I can tell `searchTerm` isn't set anywhere. Is there a place for the user to actually type in a search query somewhere?

Reply:
I somehow missed that. Fixed